veracruz-utils/src/platform/icecap/message.rs

```diff
@@ -17,14 +17,12 @@ pub type SessionId = u32;
 /// Type of requests from the host to the realm
 #[derive(Debug, Clone, Serialize, Deserialize)]
 pub enum Request {
```

Comment:
Are we now at a point where we can unify this file with `RuntimeManagerMessage` in `veracruz-utils/src/platform/vm.rs`?

Reply:
I think so, but I think there's a slightly more involved change that I want to make, namely splitting `RuntimeManagerMessage` up into a `RuntimeManagerRequest` and `RuntimeManagerResponse` type and then unifying all of the types.  I will do that next: see #374.